survey/4-ai.yml

```diff
@@ -0,0 +1,70 @@
+title: AI
+label: ai
+position: 4
+questions:
+  - label: "How often do you use AI tools for Dev (Github Copilot, ChatGPT ...) "
+    required: true
+    choices:
+      - Daily
+      - Occasionally
+      - Tried them but not interested
+      - Never
+
+  - label: "If you are using AI based coding assistants for Dev, How much improvement do you see in your productivity?"
+    choices:
+      - Not noticeable
+      - 2X - 3X
+      - 4X - 5X
+      - +5X
+
+  - label: "Do you think AI will replace developers in the future?"
+    required: true
+    choices:
+      - Not very much ( less than 10% )
+      - Somewhat ( 10% - 30% )
+      - To some extent ( 30% - 50% )
+      - The majority ( 50% - 80% )
+      - Completely  ( more than 80% )
+      - Not sure
+
+  - label: "Are you planning to learn AI to build apps?"
+    required: true
+    choices:
+      - No, I am not interested
+      - Already started learning the basics.
+      - I can build simple apps with AI using third-party APIs.
+      - I can build complex apps and play with models.
+      - I am already an expert
+
+  - label: "How much is your company investing in AI?"
+    required: true
+    choices:
+      - We haven't started adopting AI
+      - We're getting the basics in place
+      - We have some ad-hoc use cases in production
+      - We have several use cases in production
+
+  - label: "What are the tools, apps, frameworks, and models you are using?"
+    required: true
+    multiple: true
+    choices:
+      - ChatGPT
+      - PaLM (Google)
+      - Claude (Anthropic)
+      - DALL-E (OpenAI)
+      - LangChain
+      - GitHub Copilot
+      - Notion AI
+      - Coby.ai
+      - Grammarly
+      - AWS Bedrock
+      - Together.ai
+      - Weights & Biases
+      - LlamaIndex
+      - Haystack
+      - Humanloop
+      - Unstructured
+      - PyTorch
+      - TensorFlow
+      - Other
```

Comment:
Any chance we can add more questions from [this msg](https://github.com/geeksblabla/stateofdev.ma/pull/108#discussion_r1412603782)? 👀